src/ocm/components/featureSupportLevels/featureStateUtils.ts

```diff
@@ -96,7 +96,7 @@ const getLvmDisabledReason = (cluster: Cluster | undefined, isSupported: boolean
     return undefined;
   }
   if (!isSupported) {
-    return 'OpenShift Data Foundation Logical Volume Manager is enabled only for OpenShift 4.11 and above.';
```

Comment:
In this same file, there is a constant named `LVM_OPERATOR_LABEL` which needs to change its content. That's for the tooltip when the two operators are incompatible etc.

Reply:
Let's add a bug for this please. @ammont82 can you please report it.

Reply:
True, has been added in 2.13 version. I'll change to the 2.14 version